### DISCUSSION TITLE
BCrypt should be default, but optional

### DISCUSSION THREAD

**Discussion by caniko** [fastapi-users/fastapi-users#948] [Ideas]
There are other algorithms that are alternative to bcrypt such as `scrypt`.

`bcrypt` should be installed if no other algorithm is selected during install.
Installs `bcrypt`:
``` shell
pip install fastapi-users
pip install fastapi-users[tortoise-orm]
```
Installs something else instead off `brcypt`:
``` shell
pip install fastapi-users[<name_of_encryption>]
pip install fastapi-users[tortoise-orm, <name_of_encryption>]
```

**Comment by caniko** [comment]
I made a PR #950

**Comment by frankie567** [comment]
Released in [`v9.3.0`](https://github.com/fastapi-users/fastapi-users/releases/tag/v9.3.0).
